contracts/cheddar_5x5_tic_tac_toe/src/lib.rs

```diff
@@ -636,20 +636,28 @@ mod tests {
     }
 
     /// This function is used to print out the board in a human readable way
-    fn print_tiles(tiles: &[[Option<Piece>; BOARD_SIZE]; BOARD_SIZE]) {
+    fn print_tiles(tiles: Tiles) {
```

Comment:
@robert-zaremba Should this be a reference? I tried reference but it will have to be cloned inside of the function because of line `647`.

Reply:
This is a helper function in `tests` module. So we are fine here. But in general reference type is preferred.